### ISSUE TITLE
add gallery page view

### ISSUE THREAD

**Issue by QuantumPhilosophy** [QuantumPhilosophy/anonymous-pumpkin#19]
### User Story
**Nice-To-Have:** Gallery page view showing previously searched for images

### Tasks
- [x] Create `gallery.html` file
- [x] Add function to auto populate the page with previously searched for images.

## Notes
Keep in mind while coding the following:
  * **Nice-To-Have:** #15 showing only images searched by user

### Issue Links
**Depends On:** API helper setup for GoogleCloudVision #16 and Wikipedia #17
